docs/docs/cmd/spo/file/file-sharinglink-remove.md

```diff
@@ -0,0 +1,52 @@
+# spo file sharinglink remove
+
+Removes a specific sharing link to a file
+
+## Usage
+
+```sh
+m365 spo file sharinglink remove [options]
+```
+
+## Options
+
+`-u, --webUrl <webUrl>`
+: The URL of the site where the file is located.
+
+`--fileUrl [fileUrl]`
+: The server-relative (decoded) URL of the file. Specify either `fileUrl` or `fileId` but not both.
+
+`--fileId [fileId]`
+: The UniqueId (GUID) of the file. Specify either `fileUrl` or `fileId` but not both.
+
+`-i, --id <id>`
+: The ID of the sharing link.
+
+`--confirm`
+: Don't prompt for confirmation.
+
+--8<-- "docs/cmd/_global.md"
+
+## Examples
+
+Removes a specific sharing link from a file by id without prompting for confirmation.
+
+```sh
+m365 spo file sharinglink remove --webUrl https://contoso.sharepoint.com/sites/demo --fileId daebb04b-a773-4baa-b1d1-3625418e3234 --id U1BEZW1vIFZpc2l0b3Jz --confirm
```

Comment:
Also forgot to mention this, in the examples, all IDs are of this format, but when I list my sharing links, all IDs are GUIDs. Could you recheck this and if you have the same result, update the examples so the IDs are GUIDs?